plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraServer.java

```diff
@@ -122,7 +122,7 @@ private static String prepareCassandraYaml()
 
         File yamlFile = tmpDirPath.resolve("cu-cassandra.yaml").toFile();
         yamlFile.deleteOnExit();
-        write(modified, yamlFile, UTF_8);
+        writeString(yamlFile.toPath(), modified);
```

Comment:
I'd prefer UTF-8 to be explicit.
i know the `Files.writeString` uses UTF-8 by default, but providing it explicitly _also_ conveys the intent.